index.js

```diff
@@ -48,20 +51,14 @@ function update () {
 }
 
 function open () {
-  try {
-    var targetPkg = require(path.join(process.cwd(), 'package'))
-  } catch (e) {
+  var currentCoords = geopkg.getPackageCoords()
+  if (currentCoords) {
+    return geopkg.openMaps(currentCoords, _done)
+  } else {
```

Comment:
This `else` block will run both if package.json couldn't be found but also if the package.json didn't contain any coordinates - so the error shown is not always correct.

Previously the code differentiated between showing an error for when package.json wasn't found and when the package.json didn't contain any coordinates.